composer.json

```diff
@@ -13,8 +13,11 @@
     "license": "MIT",
     "authors": [
         {
+            "name": "PrestaShop SA",
+            "email": "contact@prestashop.com"
+        },{
```

Comment:
```suggestion
        },
        {
```

Reply:
It's a merge request should have be done ealier in a previous PR